docs/fluvio/tutorials/github-to-discord.mdx

```diff
@@ -149,17 +154,16 @@ An alert with `:star2: 1769` will show-up in your discord channel. See it live a
 
 ### Reference
 
-* [Create a Github Stars/Forks Event Pipeline (Blog)]
-* [labs-stars-forks-changes-sm](https://github.com/infinyon/labs-stars-forks-changes-sm)
+* [labs-stars-forks-changes-sm]
 
 
 ---
 
+[labs-stars-forks-changes-sm]: https://github.com/infinyon/labs-stars-forks-changes-sm
 [Fluvio CLI]: /docs/fluvio/quickstart#install-fluvio
```

Comment:
This is not the right format. It should be something like: `fluvio/quickstart.mdx#install-fluvio`

Reply:
Fixed!